plugins/removeHiddenElems.js

```diff
@@ -74,6 +75,21 @@ exports.fn = (root, params) => {
    */
   const removedDefIds = new Set();
 
+  /**
+   * @type {Map<XastElement, XastParent>}
+   */
+  const allDefs = new Map();
+
+  /**
+   * @type {Map<string, Array<{node: XastElement, parentNode: XastParent }>>}
```

Comment:
```suggestion
   * @type {Map<string, Array<{node: XastElement, parentNode: XastParent}>>}
```

```suggestion
   * @type {Map<string, Array<{ node: XastElement, parentNode: XastParent }>>}
```

Very petty, and not sure if the linter cares, but could we do one of these? ^-^'